pgjdbc/src/main/java/org/postgresql/core/PGStream.java

```diff
@@ -245,7 +245,11 @@ private Socket createSocket(int timeout) throws IOException {
       return socket;
     } catch ( Exception ex ) {
       if (socket != null) {
-        socket.close();
+        try {
+          socket.close();
+        } catch (Exception ex1 ) {
```

Comment:
Could you apply auto format? 
I see some style difference in spaces between line 246 and 250

Reply:
checkstyle will catch my mistakes :)

Reply:
Yeah, you're right )